### PR TITLE
Decode Reveal to UTF-8 Utility Added

### PR DESCRIPTION
**Pull request by adityanithariya**
## Motivation

Utility for decoding bigint to string is needed for smooth SDK integration, in reference to [this](https://github.com/anon-aadhaar/anon-aadhaar/issues/221) issue.

## Change Summary

Utility function for decoding BigInt to String is added.

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [ ] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [x] PR has a [change set](https://github.com/privacy-scaling-explorations/anon-aadhaar/CHANGELOG.md)
- [ ] PR has been tagged with a change label(s) (i.e. documentation, feature, bug fix, or chore)
- [ ] PR includes documentation if necessary.

## Additional Changes

PR Template change set URL correction.

### CODE REVIEW

File: packages/core/src/utils.ts [hunk 1]
```diff
@@ -98,6 +98,15 @@ export function convertBigIntToByteArray(bigInt: bigint) {
   return result.reverse()
 }
 
+export function convertCircuitBigIntToString(input: bigint) {
+  let result: string;
+  while (input > 0) {
+    result += String.fromCharCode(Number(input % BigInt(256)))
+    input = input / BigInt(256)
+  }
+  return result
```
Comment (saleel) [inline packages/core/src/utils.ts:107]: I think we need to `reverse` the `result` here.

Comment (saleel) [reply packages/core/src/utils.ts:110]: Sorry, `reverse` is not required as we are diving by 256 (which gives last char) and appending to the string.

Would be nice to add a comment like `The proof bigInt is in big endian format and each iteration appends the last char - which is the first char in little-endian format`

File: packages/core/src/utils.ts [hunk 2]
```diff
@@ -98,6 +98,18 @@ export function convertBigIntToByteArray(bigInt: bigint) {
   return result.reverse()
 }
 
+export function convertCircuitBigIntToString(input: bigint | string) {
```
Comment (saleel) [inline packages/core/src/utils.ts:101]: On second thoughts, we can call it `convertRevealBigIntToString`

Comment (adityanithariya) [reply packages/core/src/utils.ts:101]: Sure, I'll change this!

**Review by saleel — APPROVED** [review]

**Pull request merged**